frontend/setupChelonia.js

```diff
@@ -312,6 +313,7 @@ const setupChelonia = async (): Promise<*> => {
       console.error('[setupChelonia] Error syncing identity contract and groups', e)
     })
   })
+  */
```

Comment:
This section being commented out... does it make sense to either:

- Leave it uncommented, but run it only when we're sure that we're no longer running in a SW?
- Or, if it's no longer necessary at all (even when not running in a SW), perhaps it should be deleted then?

Reply:
I doesn't seem to be needed now, and while it might be needed if not running in a SW, I'm not entirely sure. I commented it out in case it's needed when that's implemented.